src/views/display/waiting.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from "react"
```

Comment:
this file was replaces by `src/views/display/waiting/waiting.tsx`. You should remove this from PR

Reply:
Will remove